spec/factories/casa_cases_emancipation_options.rb

```diff
@@ -0,0 +1,4 @@
+FactoryBot.define do
+  factory :casa_cases_emancipation_option do
+  end
+end
```

Comment:
Can you fill this in, and create a spec in `spec/models/casa_cases_emancipation_options.rb` that checks that it has a valid factory?  See `spec/models/judge_spec.rb` for an example.